src/etc/inc/plugins.inc.d/openvpn.inc

```diff
@@ -150,6 +151,25 @@ function openvpn_compression_modes()
     );
 }
 
+function openvpn_buffer_settings()
+{
+    return array(
```

Comment:
Do you have some performance metrics about the impact of this feature? In my experience sofar on FreeBSD increasing socket send and receive buffers don't impact performance as one would expect. Adding fine grained options like these only make sense if we are able to explain in which circumstances changing the options helps.

When there's a generic issue around send and receive buffers, the sysctl parameters (net.inet.tcp.recvspace, net.inet.tcp.sendspace) might be more suitable to increase (https://www.freebsd.org/cgi/man.cgi?query=tuning)

Reply:
I had a similar case where company is transfering video files heavily. At first the tunnel was set to TCP and no client was able to transfer more than 15Mbps, then we added a second instance with UDP, transfer rates grew to 25Mbps. After this we added to advanced field:
```
sndbuf 393216
rcvbuf 393216
```
Now clients are able to transfer at around 70Mbps .. which is quite the max of vdsl you get in our area. 
It was tested with 21.7 as the base, the system is not updated to 22.1 yet.